### ISSUE TITLE
Error getting RMW implementation identifier  

### ISSUE THREAD

**Issue by deepak-anzo** [ros2/rmw_cyclonedds#478]
## Bug report

 [rcl]: Error getting RMW implementation identifier / RMW implementation not installed (expected identifier of 'rmw_cyclonedds_cpp'), with error message 'failed to load shared library 'librmw_cyclonedds_cpp.so' due to dlopen error: libddsc.so.0: cannot open shared object file: No such file or directory, at ./src/shared_library.c:99, at ./src/functions.cpp:65', exiting with 1., at ./src/rcl/rmw_implementation_identifier_check.c:139 

**Required Info:**

- Operating System:
  -  OS and version  LUbuntu 22.04  ROS2 HUMBLE
- Installation type:
  - 
- Version or commit hash:
  - 
- DDS implementation:
  - 
- Client library (if applicable):
  - <!  rclcpp -->

**Comment by clalancette** [comment]
Can you give some information on what steps you followed to install Humble?

**Comment by 0116lhc** [comment]
I have the same situation,  you can try `sudo apt-get install -y ros-humble-rmw-cyclonedds-cpp `

**Comment by fujitatomoya** [comment]
@deepak-anzo can you go through https://docs.ros.org/en/humble/Installation/DDS-Implementations/Working-with-Eclipse-CycloneDDS.html? i think you are just missing the cyclone dds package.

**Comment by deepak-anzo** [comment]
I found the solution for this. 
Just put `/opt/ros/humble/lib/x86_64-linux-gnu ` in your ld config file.

**Comment by lqiang2003cn** [comment]
> I have the same situation, you can try `sudo apt-get install -y ros-humble-rmw-cyclonedds-cpp `

thank you very much, your answer saved my day. 
I might have accidentally deleted rmw while i was cleaning my system. Reinstall ros-humble-rmw-cyclonedds-cpp solved the problem.

**Comment by jdgre1** [comment]
Yes this helped me too with my wsl with humble vscode setup! Thanks!
